ledger.go

```diff
@@ -829,6 +824,8 @@ func (l *Ledger) query() {
 		votes = append(votes, vote)
 	}
 
+	wg.Wait()
```

Comment:
Can this actually ever wait for anything ?  It seems like the only thing it could wait on is for all the goroutines to finish, which the last thing they do is write to the channel which we consumed just prior to this

Reply:
actually you're right. since we read for certain number of values from channel we don't need wait group here at all